### ISSUE TITLE
Can I contribute to pytorch-forecasting

### ISSUE THREAD

**Issue by wangbingnan136** [jdb78/pytorch-forecasting#537]
Hello,sir.I am very interested in pytorch-forecasting after  I investigated all the python implementations of time series forecasting.May I participate in this project?

**Comment by jdb78** [comment]
Yes, of course! Here are some guidelines for contribution: https://pytorch-forecasting.readthedocs.io/en/latest/contribute.html

Best is to propose the feature in an issue before raising a PR. This will prevent you from developing something that will not be merged.
